### PR TITLE
 535 export to csv is missing

### PR DESCRIPTION
**Pull request by ubbabeck**
# About PR

This pr is going to add the ability for the user to export their payment history to a csv file.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -109,6 +110,10 @@ dependency_overrides:
   intl: ^0.18.1
   path: ^1.8.3
   test_api: <0.5.0 # Requires Flutter 3.10
+  fimber: ^0.6.6
```
Comment (erdemyerebasmaz) [inline pubspec.yaml:113]: We already declare fimber plugins under `dependencies:`

File: pubspec.yaml [hunk 2]
```diff
@@ -94,6 +94,7 @@ dependencies:
   workmanager:
     git:
       url: https://github.com/breez/flutter_workmanager.git
+  csv: ^5.0.2
```
Comment (erdemyerebasmaz) [inline pubspec.yaml:97]: dependencies ares alphabetically ordered, please put this after `clipboard_watcher`

**Review by erdemyerebasmaz — COMMENTED** [review]
Two nit comments regarding dependencies, would be great to see a video before we approve & merge.

File: pubspec.yaml (hunk 1 above)
Comment (ubbabeck) [reply pubspec.yaml:113]: I had to override them in order to get the logs for iOS. Does the logging work in production with the latest fimber version.

Comment (ubbabeck) [reply pubspec.yaml]: Done

File: pubspec.yaml (hunk 2 above)
Comment (ubbabeck) [reply pubspec.yaml]: Thanks for letting me know. Done.

**Comment by ubbabeck** [comment]
@erdemyerebasmaz here is a example of how it looks. :D 


https://github.com/breez/c-breez/assets/36157890/5fe9afea-a2f4-41a2-8fc5-45ec3968d5ba

**Review by erdemyerebasmaz — APPROVED** [review]

**Review by roeierez — APPROVED** [review]

**Pull request merged**